### ISSUE TITLE
Question about training loss

### ISSUE THREAD

**Issue by YS-Kuang** [open-mmlab/OpenPCDet#990]
If i freeze the parameters of the first stage and train the second stage，do i still need to combine the loss of these two stages?
